haskell/quicksort.hs

```diff
@@ -0,0 +1,13 @@
+quicksort :: (Ord a) => [a] -> [a]  
+quicksort [] = []  
+quicksort (x:xs) =   
+    let smaller = quicksort [a | a <- xs, a <= x]  
+        bigger = quicksort [a | a <- xs, a > x]  
+    in  smaller ++ [x] ++ bigger 
+
+
+main = do 
+    print(quicksort [7,15,12,-5,14,88,52,11])
```

Comment:
Please use assertion with sorted array in separate function and call that in main